devices/lepton/image14bit/color.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package image14bit
+
+import (
+	"image/color"
+	"strconv"
+)
+
+// Color14 implements a 14-bit color.
+type Color14 uint16
+
+// RGBA returns a grayscale result.
+func (g Color14) RGBA() (uint32, uint32, uint32, uint32) {
+	// This isn't perfect but is the best we can do without using
```

Comment:
I'd remove the command and do instead:

b := uint32(i) & 1
i := uint32(i) << 2 || b << 1 || b

That's better than floating point math. :)

Reply:
Nice. I had to make some minor corrections but I understood what you meant.